estate/models/estate_property.py

```diff
@@ -66,3 +67,38 @@ class EstateProperty(models.Model):
         "property_id",
         string="Offers",
     )
+    total_area = fields.Float(compute="_compute_total_area", string="Total Area (sqm)")
+    best_price = fields.Float(compute="_compute_best_prince", string="Best Offer")
+
+    @api.depends("living_area", "garden_area")
+    def _compute_total_area(self):
+        for record in self:
+            record.total_area = record.living_area + record.garden_area
+
+    @api.depends("offer_ids.price")
+    def _compute_best_prince(self):
+        for record in self:
+            offer_prices = record.offer_ids.mapped("price")
+            if not offer_prices:
+                record.best_price = 0
+            else:
+                record.best_price = max(offer_prices)
+
+    @api.onchange("garden")
+    def _onchange_garden_(self):
+        self.garden_area = 10
+        self.garden_orientation = "north"
+
+    def action_sold_property(self):
+        for record in self:
+            if record.status == "canceled":
+                raise UserError(_("A canceled property can not be sold"))
+            record.status = "sold"
+        return True
+
+    def action_cancel_property(self):
+        for record in self:
+            if record.status == "sold":
+                raise UserError(_("A sold property can not be canceled"))
```

Comment:
Why `_()`? Do you know what that method does?

```suggestion
                raise UserError(_("A sold property can not be canceled."))
```